src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Base/base.props

```diff
@@ -10,5 +10,8 @@
 				XamlRuntime="WinUI"
 				DependentUpon="AppHead.xaml"
 				Link="AppHead.xaml.cs" />
+		<UnoIcon Include="$(MSBuildThisFileDirectory)Icons\iconapp.svg"
+				ForegroundFile="$(MSBuildThisFileDirectory)Icons\appconfig.svg"
+				Color="#00000000" />
```

Comment:
If I'm not mistaken this will make the iconapp.svg show up in the root of each head project. Let's add `LinkBase="AppIcons"` to fix that.

Reply:
I'll give it a try from the template on this PR but in the resizetizer sample and when I ran a test locally the files didn't appear